client/trino-jdbc/pom.xml

```diff
@@ -91,6 +86,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-context</artifactId>
+            <scope>provided</scope>
```

Comment:
This essentially means that we assume that these two dependencies and all transitive dependencies are available at runtime of the jdbc driver. If they're not available opentelemetry will not work. While that's fine I think we need to document what dependencies uses will need to add if they want to have open telemetry integration from each other client that uses the driver into Trino and beyond.

Reply:
No. They are optional and needed only if you want to use TracingDriver